### PR TITLE
Readds blob cuddling

### PR DESCRIPTION
**Pull request by ghost**
except this time the blob can toggle if they wanna see the messages or not

:cl: More Robust Than You
add: Readds blob cuddling
add: Blob Overminds can now toggle their CuddleSense!
/:cl:

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
aaaaa uhh ohoo?

**Comment by PraiseRatvar** [comment]
fucked that review uph aa

**Comment by lzimann** [comment]
this is not happening regardless of toggle

**Comment by Ratvar** [comment]
imageMoreRobustThanYou:
add: Readds voredogs!
add: Players can now toggle their VorePreferences!
/image

File: code/game/gamemodes/blob/overmind.dm [hunk 1]
```diff
@@ -26,6 +26,7 @@
 	var/base_point_rate = 2 //for blob core placement
 	var/manualplace_min_time = 600 //in deciseconds //a minute, to get bearings
 	var/autoplace_max_time = 3600 //six minutes, as long as should be needed
+	var/acceptLove = TRUE
```
Comment (ChangelingRain) [inline code/game/gamemodes/blob/overmind.dm]: Default to off.

**Comment by FantasticFwoosh** [comment]
⛔️

**Comment by Cyberboss** [comment]
Even I was against the original removal, but stop baiting

**Comment by Cyberboss** [comment]
Duplicate of #30008

**Comment by Arzion5** [comment]
Can we please replace the maintainers with clowns? At least clowns know what is fun.

**Pull request closed without merging**